### DISCUSSION TITLE
Custom Form in Custom Page ?

### DISCUSSION THREAD

**Discussion by kvostyc** [filamentphp/filament#6399] [Help] [answered]
Heyy, is there any way how to make my own form component and insert it to my custom page view ? Im trying to kinda find it on documentation, but im really desperate rn. Maybe I just need a little push with providing correct links to docs, if theres are any

Thanks in advance,

**Comment by danharrin** [comment, marked as answer]
Custom pages are Livewire components. You can follow these instructions - https://filamentphp.com/docs/2.x/forms/getting-started#preparing-your-livewire-component
